SConstruct

```diff
@@ -170,23 +170,68 @@ opts = Variables(customs, ARGUMENTS)
 # Target build options
 opts.Add("platform", "Target platform (%s)" % ("|".join(platform_list),), "")
 opts.Add("p", "Platform (alias for 'platform')", "")
-opts.Add(EnumVariable("target", "Compilation target", "editor", ("editor", "template_release", "template_debug")))
-opts.Add(EnumVariable("arch", "CPU architecture", "auto", ["auto"] + architectures, architecture_aliases))
+opts.Add(
+    EnumVariable(
+        "target",
+        "Compilation target",
+        "editor",
+        ("editor", "template_release", "template_debug"),
+    )
+)
+opts.Add(
+    EnumVariable(
+        "arch",
+        "CPU architecture",
+        "auto",
+        ["auto"] + architectures,
+        architecture_aliases,
+    )
+)
 opts.Add(BoolVariable("dev_build", "Developer build with dev-only debugging code (DEV_ENABLED)", False))
 opts.Add(
     EnumVariable(
-        "optimize", "Optimization level", "speed_trace", ("none", "custom", "debug", "speed", "speed_trace", "size")
+        "optimize",
+        "Optimization level",
+        "speed_trace",
+        ("none", "custom", "debug", "speed", "speed_trace", "size"),
     )
 )
 opts.Add(BoolVariable("debug_symbols", "Build with debugging symbols", False))
 opts.Add(BoolVariable("separate_debug_symbols", "Extract debugging symbols to a separate file", False))
-opts.Add(EnumVariable("lto", "Link-time optimization (production builds)", "none", ("none", "auto", "thin", "full")))
+opts.Add(
+    EnumVariable(
+        "lto",
+        "Link-time optimization (production builds)",
+        "none",
+        ("none", "auto", "thin", "full"),
+    )
+)
 opts.Add(BoolVariable("production", "Set defaults to build Godot for use in production", False))
-opts.Add(BoolVariable("generate_apk", "Generate an APK/AAB after building Android library by calling Gradle", False))
+opts.Add(
+    BoolVariable(
+        "generate_apk",
+        "Generate an APK/AAB after building Android library by calling Gradle",
+        False,
+    )
+)
+opts.Add(BoolVariable("use_threads", "Enable threading support", True))
```

Comment:
I wanted to test this PR and intuitively I used `scons threads=no`. We still have a couple `use_<thing>` options but I think for most we're now using just `<thing>=yes|no`.

WDYT? I don't feel too strongly about this either way. Just noticed that `threads=no` came more naturally to me.

Reply:
Renamed to `threads=yes|no`!